### ISSUE TITLE
Support for pre-composed handles

### ISSUE THREAD

**Issue by BCDigLib** [projecthydra-labs/hybox-ideas#15]
At Boston College we assign handles to our digital objects before they are created.  This is because the handle's suffix re-uses a unique identifier from another system such as our archival management system.  

We need to be able to include these pre-composed handles in our ingest packages rather than being tied to handles that hybox would assign as part of a sequence. 

For example, we assign a component unique identifier to a resource, BC1986_032_ref101,
which is mapped to OBJID in a METS document, resulting in the following handle generated on ingest

http://hdl.handle.net/2345.2/BC1986_032_ref101

Currently we ingest our handles using PREMIS in METS and would need a standards based place to store handles for ingest to HyBox.

**Comment by jcoyne** [comment]
Would you have objects that get ingested that don't have handles pre-generated, that would need handles generated?  Should this support Handle.net, ezid, purl.standford.edu, doi, all of the above?  Would you need to update a handle to point a new url as exposed in Hybox?

**Comment by BCDigLib** [comment]
Yes, we could have objects without pre-generated handles - being able to define a syntax might be nice when a handle needs to be generated (say adding a prefix to the PID of the object)

We're mostly interested in handle.net and doi. Being able to update a handle would be nice as well - currently when I need to do this I just use the handle client.
